tests/pages/bigtest/settings.js

```diff
@@ -0,0 +1,38 @@
+import {
+  attribute,
+  blurrable,
+  clickable,
+  computed,
+  fillable,
+  isPresent,
+  page,
+  property,
+  text,
+  value
+} from '@bigtest/interaction';
+
+let hasClassBeginningWith = (className, selector) => {
+  return computed(function () {
+    return this.$(selector).className.includes(className);
+  });
+};
```

Comment:
🔥

Reply:
This was easier than the options discussed in https://github.com/thefrontside/bigtest/issues/52, since the page object doesn't have to pass through the webpack `style-loader`.